CHANGELOG.md

```diff
@@ -13,7 +13,7 @@
 -----------
 
 ### Internals
-* None.
+* Releases for Apple platforms are now built with Xcode 12.2.
```

Comment:
Xcode 12.4 on EVG actually.

Reply:
The release package isn't built on evergreen.